### PR TITLE
Add bio for jayg

### LINKED ISSUE #23: Add a bio for jayg
- [x] Create a branch
- [x] Create a jayg.md
- [x] Open a pull request
- [x] Get and receive feedback
- [x] Merge the pull request

### PR DESCRIPTION
**Pull request by jaygarala**
Fixes #23 Bio for Jayg

### CODE REVIEW

File: jaygarala.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# GitHub Student
+
+* Name: Jay Garala
+* Username: jaygarala
+* Location: Reston, VA
```
Comment (ChrisJergens) [inline jaygarala.md:5]: @jaygarala can you add a little more detail?

Comment (jaygarala) [inline jaygarala.md:5]: @ChrisJergens cool now?

**Comment by jaygarala** [comment]
@ChrisJergens Added Team

**Comment by jaygarala** [comment]
:helicopter:

**Pull request merged**